spec/controllers/organizations_controller_spec.rb

```diff
@@ -1,14 +1,61 @@
 require 'spec_helper'
 
 describe OrganizationsController do
-  describe '#show' do
-    let(:organization) { Fabricate(:organization) }
-
-    it 'links to new_transfer_path' do
-      get 'show', id: organization.id
-      expect(response.body).to include(
-        "<a href=\"/transfers/new?destination_account_id=#{organization.account.id}&amp;id=#{organization.id}\">"
-      )
+  let!(:organization) { Fabricate(:organization) }
+  let(:member) { Fabricate(:member, organization: organization) }
+
+  describe 'GET #show' do
+    context 'with a logged user (organization member)' do
+      it 'links to new_transfer_path' do
+        login(member.user)
+
+        get 'show', id: organization.id
+        expect(response.body).to include(
+          "<a href=\"/transfers/new?destination_account_id=#{organization.account.id}&amp;id=#{organization.id}\">"
```

Comment:
IMO views specs are better suited for this. We started doing it like this and when we started feeling the pain we switched to views specs.

Reply:
Totally agree! view specs are better option to test those kind of things